docs/CONFIGURATION.md

```diff
@@ -50,11 +50,13 @@ When using the sbt plugin, this configuration is ignored and the sbt test runner
 
 #### reporters
 
-**Config file:** `reporters: ["console"]`  
-**Default value:** A reporter that will report to console.  
+**Config file:** `reporters: ["console", "html"]`  
+**Default value:** A reporter that will report to `console`.  
 **Mandatory:** No  
 **Description:**  
 With `reporters` you can specify reporters for stryker4s to use. By default the `console` reporter is used which will report to your console.
+The `html` reporter is reporter that will generate a web page based report giving a full overview of the mutation run.
+
```

Comment:
Might be nice to link [the example report from the stryker site](https://stryker-mutator.io/robobar-example/reports/mutation/html/index.html) here? 🙂

Reply:
That report is the (older) Stryker report which is different from the newer web-components based one. Once that has a new one I agree we should link to it though!